README.md

```diff
@@ -47,9 +47,14 @@ Note this starts up all required services, including Redis, MySQL, and Memcached
 
 Use the [PM2 tool](https://github.com/Unitech/PM2#main-features) to stop and start the servers, and read server logs.
 
+To install pm2:
+- `npm install pm2 -g`
+
 To start all servers:
 
 - `npm start`
+- `./pm2 start mysql_servers.json`
```

Comment:
I tested this locally and `npm start` started all the servers. I think you remove this part.

Reply:
okay, sure. Thank you!